vassal-app/src/main/java/VASSAL/tools/swing/SwingUtils.java

```diff
@@ -201,7 +201,7 @@ public boolean isSelectionToggle(MouseEvent e) {
     @Override
     @SuppressWarnings("deprecation")
     public KeyStroke systemToGeneric (KeyStroke k) {
-      if (macLegacy) {
+      if (macLegacy || (k == null)) {
```

Comment:
I had a look at where these are used. Nothing should be passing nulls to these. The original call in `KeyStrokeListener` had an implicit null check because we were calling `stroke.equals(key)` and `stroke` had already been verified to be nonnull. We can omit the null checks here.

Reply:
I had made a similar tour and concluded that nonetheless some "future-proofing" was reasonable, but I've removed those two extra checks in the new push.